DigitalLearningSolutions.Data/DataServices/GroupsDataService.cs

```diff
@@ -187,18 +187,22 @@ public IEnumerable<GroupDelegate> GetGroupDelegates(int groupId)
         {
             return connection.Query<GroupDelegate>(
                 @"SELECT
-                        GroupDelegateID,
-                        GroupID,
-                        DelegateID,
-                        FirstName,
-                        LastName,
-                        EmailAddress,
-                        CandidateNumber,
-                        AddedDate,
-                        HasBeenPromptedForPrn,
-                        ProfessionalRegistrationNumber
+                        gd.GroupDelegateID,
+                        gd.GroupID,
+                        gd.DelegateID,
+                        gd.AddedDate,
+                        da.CandidateNumber,
+                        u.FirstName,
+                        u.LastName,
+                        u.HasBeenPromptedForPrn,
+                        u.ProfessionalRegistrationNumber,
+                        u.PrimaryEmail,

```

Comment:
After a second look, I realised here I did something similar for 897, but instead of selecting both primary email and centre email I did `COALESCE(ucd.Email, u.PrimaryEmail)` and I kept the `GroupDelegate` class the same. The primary and centre emails are used together as `centreEmail ?? primaryEmail` everywhere, and basically I did this operation directly on the database. Not sure which approach is better.

Reply:
It depends on whether we're wanting to email unverified centre emails or not. For 919, this is the case according to the ticket, so I needed the all the details to be pulled through (and didn't want to cause horrible duplicate logic in SQL code). 

It looks like the criteria for 897 is to display always, so where we're not sending emails, COALESCE is probably fine, otherwise do it in c#.

Reply:
That makes sense, thanks!